source/content/filezilla.md

```diff
@@ -80,6 +82,15 @@ Resolve instances of transfer failures by reconfiguring FileZilla to limit the n
 
 See also, [this FAQ on our related SFTP doc](/sftp#i-am-receiving-errors-connecting-to-my-server-with-an-sftp-client).
 
+#### Fallback solution when file transfer fails
+If all else fails, you can connect directly to an application container via the IP address. Use Dig to find the IP address:
+
+```bash{outputLines:2-3}
+dig +short appserver.live.120330a1-xxxxxxxxxxxxxxxxx.drush.
+35.194.x.x
+35.222.x.x
```

Comment:
@alexfornuto should these be changed to example IPs or are the results always going to be in that IP range? If the numbers here are consistent, then I'm 👍

Reply:
@carl-alberto do all Pantheon IPs fall into these two class Bs?